source/_integrations/hunterdouglas_powerview.markdown

```diff
@@ -34,6 +34,12 @@ There is currently support for the following device types within Home Assistant:
 - Scene
 - Sensor
 
+<div class="note">
+The Powerview Hub does not automatically wake shades or report position changes made via pebble remotes.
+
+Calling the update entity service (`homeassistant.update_entity`) on a shade entity will trigger the hub to awaken a shade and report its current position.  [An example automation is available](#force-update-shade-position) below for mains powered shades.  While the automation will work for battery-powered shades, it will quickly drain their batteries for these devices.
```

Comment:
```suggestion
Calling the update entity service (`homeassistant.update_entity`) on a shade entity will trigger the hub to awaken a shade and report its current position. [An example automation is available](#force-update-shade-position) below for mains powered shades. While the automation will work for battery-powered shades, it will quickly drain their batteries for these devices.
```